plugin/ToolBar.qml

```diff
@@ -22,49 +22,39 @@ import Sailfish.Silica 1.0
 PanelBackground {
     id: toolbar
 
-    property var flickable: undefined
-    property int parentHeight
-    property bool hidden
+    property Item flickable
+    property bool forceHidden
     property bool autoHide: true
+    property bool trackMovement: true
```

Comment:
Didn't test, but from reading it seems to me that trackMovement is redundant with autoHide because in PDFDocumentPage.qml trackMovement is bind to autoHide. But besides, in a generic case the two properties still have sense : we want to appear so trackMovement is true, but don't want to disappear after, so autohide is false.

Maybe this case is too generic and Toolbar.qml may be even further simplified ?